pywinauto/controls/uia_controls.py

```diff
@@ -131,3 +135,101 @@ def is_selected(self):
         elem = self.element_info.element
         iface = uia_defs.get_elem_interface(elem, "SelectionItem")
         return iface.CurrentIsSelected
+
+#====================================================================
+class ComboBoxWrapper(UIAWrapper.UIAWrapper):
+    """Wrap a UIA CoboBox control"""
+
+    control_types = [
+        IUIA().UIA_dll.UIA_ComboBoxControlTypeId
+        ]
+
+    #-----------------------------------------------------------
+    def __init__(self, hwnd):
+        """Initialize the control"""
+        super(ComboBoxWrapper, self).__init__(hwnd)
+
+    #-----------------------------------------------------------
+    def texts(self):
+        """Return the text of the items in the combobox"""
+
```

Comment:
(minor) Sorry again, but Landscape recommends to remove an empty line after a function/method doc string.